neo/Ledger/MemoryPool.cs

```diff
@@ -99,9 +100,13 @@ public MemoryPool(NeoSystem system, int capacity)
             Capacity = capacity;
         }
 
-        internal void LoadPolicy(Snapshot snapshot)
+        internal bool LoadPolicy(Snapshot snapshot)
         {
             _maxTxPerBlock = (int)NativeContract.Policy.GetMaxTransactionsPerBlock(snapshot);
+            long newFeePerByte = NativeContract.Policy.GetFeePerByte(snapshot);
+            bool policyChanged = newFeePerByte > _feePerByte;
```

Comment:
`newFeePerByte > _feePerByte` ?

Just different is enough, no?

`newFeePerByte != _feePerByte`

Reply:
Haha shargon discussed the same some time ago... only bigger prices may break tx.

Reply:
that is true, brother. But why not verify all, see my comment below.